setup.cfg

```diff
@@ -61,6 +61,8 @@ tests_require =
 zip_safe = False
 
 [options.extras_require]
+theme =
+    libsass>=0.23.0
```

Comment:
If it's possible, probably worth a comment mentioning that we repeat `libsass` in dev  as well (so if we change it here, we also need to change it there)